app/src/main/java/org/wikipedia/analytics/eventplatform/WatchlistAnalyticsHelper.kt

```diff
@@ -0,0 +1,51 @@
+package org.wikipedia.analytics.eventplatform
+
+import android.content.Context
+import org.wikipedia.WikipediaApp
+
+object WatchlistAnalyticsHelper {
+    fun logAddedToWatchlist(context: Context, itemWiki: String) {
```

Comment:
The `context` parameter for most of these calls is always `WikipediaApp.instance`, so it can become an optional parameter.

Reply:
The dynamic context parameter catches the screen name correctly in case of activity from PageFragment @dbrant .. this will help distinguish between `add` actions